definitions/features/katello_service.rb

```diff
@@ -0,0 +1,28 @@
+class Features::KatelloService < ForemanMaintain::Feature
+  metadata do
+    label :katello_service
+  end
+
+  def make_stop(spinner, args = '')
+    originally_running_services = running_services
+    spinner.update 'Stopping katello running services..'
+    execute!("katello-service stop #{args}")
```

Comment:
@iNecas ,
Yes, it will be better. As we have `running_services` list value, we can include `:only` and `:exclude` option keys.
I will modify and update the PR.